taipy/core/config/job_config.py

```diff
@@ -117,14 +119,12 @@ def is_development(self) -> bool:
         """True if the config is set to development mode"""
         return self.mode == self._DEVELOPMENT_MODE
 
-    @classmethod
-    def get_default_config(cls, mode: str) -> Dict[str, Any]:
-        if cls.is_standalone:  # type: ignore
-            return {"max_nb_of_workers": 1}
-        if cls.is_development:
+    def get_default_config(self, mode: str) -> Dict[str, Any]:
+        if self.is_standalone:
+            return {"max_nb_of_workers": self._DEFAULT_MAX_NB_OF_WORKERS}
+        if self.is_development:
```

Comment:
If this is a class method, the `cls.is_standalone` will always return True since it's a method.

To return the `is_standalone` property, it needs to be `self.is_standalone`.

Due to this bug, the `max_nb_of_workers` also exists even when the mode is DEVELOPMENT.

Reply:
Indeed.